src/interface/src/styleguide/search-bar/search-bar.component.ts

```diff
@@ -99,12 +102,6 @@ export class SearchBarComponent implements OnInit, OnDestroy {
       .subscribe((searchTerm: string) => {
         this.searchString.emit(searchTerm);
       });
-    //responds to events that request we clear the input value
-    if (this.clearEvent) {
```

Comment:
this was causing the issue, it was triggered when you open the modal.
Instead of a clear event, I think we can just clear with reseting the search term

Reply:
Weird. My goal was to replace this, too, so I tried a few versions with this gone and I was still seeing issues. But I may have done something wrong in one of those attempts, and it seems to be working, so 🤷 Thanks for the fix.